src/mon/PGMonitor.cc

```diff
@@ -1375,8 +1375,11 @@ void PGMonitor::dump_object_stat_sum(TextTable &tbl, Formatter *f,
     tbl << stringify(si_t(sum.num_bytes));
     int64_t kb_used = SHIFT_ROUND_UP(sum.num_bytes, 10);
     float used = 0.0;
-    if (pg_map.osd_sum.kb > 0)
-      used = (float)kb_used * raw_used_rate * curr_object_copies_rate / pg_map.osd_sum.kb;
+    if (avail <= sum.num_bytes / 99.9) {
```

Comment:
I think this can be simplified to avail < 1.  As long as avail is non-zero, all values of sum.num_bytes are safe.